src/main/java/com/demo/classroom/Entity/Teacher.java

```diff
@@ -0,0 +1,32 @@
+package com.demo.classroom.Entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Lob;
+import jakarta.persistence.OneToOne;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Teacher {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long teacherId;
+
+    @Lob  
+    private byte[] image;
```

Comment:
Instead of storing image in the db consider keeping it in a bucket and store the url in the db.

Reply:
solved.